### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by bughunterp0** [subfinder/subfinder#180]
===============================================
-=Subfinder v1.1.3 github.com/subfinder/subfinder
===============================================

Running Source: Ask
Running Source: Archive.is
Running Source: Baidu
Running Source: Bing
Running Source: CertDB
Running Source: CertificateTransparency
Running Source: Certspotter
Running Source: Commoncrawl
Running Source: Crt.sh
Running Source: Dnsdb
Running Source: DNSDumpster
Running Source: DNSTable
Running Source: Dogpile
Running Source: Exalead
Running Source: Findsubdomains
Running Source: Googleter
Running Source: Hackertarget
Running Source: Ipv4Info
Running Source: PTRArchive
Running Source: Sitedossier
Running Source: Threatcrowd
Running Source: ThreatMiner
Running Source: WaybackArchive
Running Source: Yahoo

Found Wildcard DNS at freelancer.com
 - 18.206.9.77
 - 54.88.15.151
 - 34.193.180.66
Running enumeration on freelancer.com

dnsdb: Unexpected return status 404
2019/10/20 17:11:34 panic while running job: runtime error: index out of range
panic: runtime error: index out of range [recovered]
	panic: interface conversion: interface {} is runtime.errorString, not string

goroutine 10 [running]:
github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).subworker.func1(0xc42009a500)
	/home/bughunter/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:61 +0x163
panic(0x70dfc0, 0x938a60)
	/usr/lib/go-1.10/src/runtime/panic.go:502 +0x229
github.com/subfinder/subfinder/libsubfinder/sources/dnsdumpster.Query(0xc42013c440, 0x2, 0x2, 0x682601, 0x101)
	/home/bughunter/go/src/github.com/subfinder/subfinder/libsubfinder/sources/dnsdumpster/dnsdumpster.go:61 +0x11f6
github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).subworker(0xc4201ce000, 0xc42009a500)
	/home/bughunter/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:64 +0x6e
github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).worker(0xc4201ce000, 0x4)
	/home/bughunter/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:78 +0xb3
created by github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).Run
	/home/bughunter/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:181 +0x7f

**Comment by averonesis** [comment]
the same

**Comment by bughunterp0** [comment]
Waiting for solution

**Comment by bughunterp0** [comment]
Remove every thing. And Update Go . Install subfinder again it rocks

**Comment by shamrocksu88** [comment]
> Remove every thing. And Update Go . Install subfinder again it rocks

Tried that didn't help. Would be great if someone could help out on this. Am using the latest version of Subfinder and Go. 
cc: @Ice3man543

**Comment by Ice3man543** [comment]
Hey, it's been fixed in the latest version! Try go get again!

**Comment by shamrocksu88** [comment]
@Ice3man543 Looks fixed now, appreciate it. Tried with binary as well as build. 

One thing I noticed is looks like most of the time HTTP timeout is stopping the scan instead of just skipping the source.

Example:

[WRN] Could not run source digicert: Get https://ssltools.digicert.com/chainTester/webservice/ctsearch/search?keyword=hackerone.com: net/http: request canceled 

After this, scan stops doesn't move to next service.
